iosApp/iosApp/LaunchView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LaunchView.swift
+//  iosApp
+//
+//  Created by Nasir Ahmed Momin on 20/05/22.
+//  Copyright © 2022 Mutual Mobile. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct LaunchView: View {
+    
+    @State private var selection = 0
+    
+    private var textList = [
+        "Simple, powerful time tracking, reporting and invoicing",
+        "Track time easily, wherever you are",
+        "Enter expanses on go",
+        "Report on time for powerful insights"
+    ]
+    
+    var body: some View {
+        
+        GeometryReader { proxy in
+            VStack {
+                Spacer().frame(height: 80.0)
```

Comment:
frame height should be a ratio of screen size, as in small devices it might not look good.

Reply:
Done, Now 10% is used instead of 80.0